src/dataflow/src/render/context.rs

```diff
@@ -65,6 +71,23 @@ where
     T: Timestamp + Lattice,
     S::Timestamp: Lattice + Refines<T>,
 {
+    /// The debug name of the dataflow associated with this context.
+    pub debug_name: String,
+    /// When set, `as_of` indicates a frontier that can be used to compact input
```

Comment:
This may have gone from being an `Option` to being mandatory, but can probably now ditch the "When set,"

Reply:
Ooh good point, done.